nifi-registry/nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/provider/flow/git/GitFlowMetaData.java

```diff
@@ -522,7 +524,8 @@ void commit(String author, String message, Bucket bucket, Flow.FlowPointer flowP
 
     byte[] getContent(String objectId) throws IOException {
         final ObjectId flowSnapshotObjectId = gitRepo.resolve(objectId);
-        return gitRepo.newObjectReader().open(flowSnapshotObjectId).getBytes();
+        ObjectStream objStream = gitRepo.newObjectReader().open(flowSnapshotObjectId).openStream();
```

Comment:
`objectStream` could be declared `final` (to stay consistent with the line above)

Reply:
yes it could, Thank you. Well. I'm headed back to the contributor guide, to see what the git flow is on a PR.

Reply:
Hmmm... May there be another validation to do? you marked this as resolved, but you still show up as pending review...